src/backend.rs

```diff
@@ -139,33 +139,29 @@ impl Backend {
         let kind = Some(CRYPTO_KIND_VLD0);
     
         let dht_record = routing_context.create_dht_record(schema, kind).await?;
-        let record_key = dht_record.key().clone();
         let keypair = vld0_generate_keypair();
         let crypto_system = CryptoSystemVLD0::new(veilid.crypto()?);
     
         let encryption_key = crypto_system.random_shared_secret();
     
         let group = Group::new(
-            keypair.key.clone(),
-            record_key,
             dht_record,
             encryption_key,
-            Some(CryptoTyped::new(CRYPTO_KIND_VLD0, keypair.secret)),
             Arc::new(routing_context),
             crypto_system,
         );
     
         let protected_store = veilid.protected_store().unwrap();
         CommonKeypair {
```

Comment:
This struct should have the id under a new `id` field and the public_key should be the owner public key.

Reply:
Just to be sure which struct we are talking about. Are you saying I should add a new `id` field to the group struct?

Reply:
To the commonkeypair struct

Reply:
- Included the id field to the keypair struct